test/samples/timestamp/input.json

```diff
@@ -1,11 +1,15 @@
 {
-	"id": "http://yourdomain.com/schemas/myschema.json",
-	"description": "Example description",
-	"type": "object",
-	"properties": {
-		"modified": {
-			"type": "string",
-			"format": "date-time"
-		}
-	}
+    "id": "http://yourdomain.com/schemas/myschema.json",
+    "description": "Example description",
+    "allOf": [
```

Comment:
We dont need to add allOf here as we have other examples that do this, can we revert this please.